clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/BaseGoogleInstanceDescription.groovy

```diff
@@ -43,6 +43,9 @@ class BaseGoogleInstanceDescription extends AbstractGoogleCredentialsDescription
   Boolean preemptible
   Boolean automaticRestart
   OnHostMaintenance onHostMaintenance
+  Boolean enableSecureBoot;
```

Comment:
Can you drop a comment block to describe what each of these are designating?

Reply:
Will do.